BraveRewards/Ledger/BATBraveLedger.mm

```diff
@@ -34,6 +34,13 @@ + (__type)__objc_getter { return ledger::__cpp_var; } \
 + (void)__objc_setter:(__type)newValue { ledger::__cpp_var = newValue; }
 
 NSString * const BATBraveLedgerErrorDomain = @"BATBraveLedgerErrorDomain";
+NSNotificationName const BATBraveLedgerNotificationAdded = @"BATBraveLedgerNotificationAdded";
+
+static NSString * const kNextAddFundsNotificationKey = @"BATNextAddFundsNotification";
```

Comment:
rename to `kNextAddFundsDateNotificationKey` to make it clear that it is a date we store here